worker/src/zimfarm_worker/task/worker.py

```diff
@@ -104,10 +105,21 @@ def __init__(
         self.dns: list[str] | None = None  # list of DNS IPs or None
         self.monitor: Container | None = None  # monitor container
 
-        self.zim_files: dict[str, dict[str, str]] = {}  # ZIM files registry
-        self.zim_retries: dict[str, int] = {}  # ZIM files with upload errors (registry)
-        self.uploader: Container | None = None  # zim-files uploader container
-        self.checker: Container | None = None  # zim-files uploader container
+        self.zim_files_actions_status: dict[str, dict[str, str]] = (
+            {}
+        )  # ZIM files registry
+        self.zim_upload_retries: defaultdict[str, int] = defaultdict(
+            int
+        )  # ZIM files with upload errors (registry)
+        self.zim_uploader: Container | None = None  # zim-files uploader container
+        self.checker: Container | None = None  # zim-files checker container
+
+        self.zimcheck_uploader: Container | None = None  # zimcheck result uploader
+        # mapping of zimcheck result filenames to zim filename. This allows multiple
+        # checkers to run independent of the rate that upload for the check results
```

Comment:
I don't get this. We still have only one `self.checker` and one `self.zimcheck_uploader`, so we will not have multiple checkers runing at the same time

Reply:
I'm totally fine with the variable name, intent, usage... global worker behavior seems ok to me.

My only concern is the comment which is very misleading. AFAIK the map contains only the check files which have completed, and does not allow to run multiple checkers or uploaders at once.